client/src/redux/reducers/workbench.reducers.js

```diff
@@ -1,12 +1,13 @@
 export const initialState = {
-  status: null
+  debug: false,
+  setupComplete: window.__INITIAL_SETUP_COMPLETE__
```

Comment:
Here's a thought - we require rendering a loading component for the workbench. if we implement that, we could have a saga that checks the loading state of required substates and then shows the workbench once we "know" that the setupComplete state has been retrieved. I don't know if this is within scope of this PR, but it definitely solves the issue of having this nasty _window variable.